index.html

```diff
@@ -135,27 +135,10 @@
             the <a>WoT Core Profile</a> can interoperate with each other
             out-of-the-box.
         </p>
-        <p>A later version of this document will define additional
-            profiles:</p>
-        <ul>
-            <li><strong>Thing Template Profile</strong>
-                <p>
-                    The <a>Thing Template Profile</a> enables to apply
-                    the vocabulary of the <a>Thing Description</a>
-                    Specification for defining a common interface, that
-                    can be implemented by different things. It selects
-                    an appropriate subset to describe <strong>classes</strong>
-                    of things that share the same data model.
-                </p></li>
-
-            <li><strong>Digital Twin Profile</strong>
-                <p>
-                    The <a>Digital Twin Profile</a> allows defining a
-                    virtual model of a thing or a group of things.
-                </p></li>
-        </ul>
-        <p>Profiles are is not exclusive, they can overlap or
-            include others.
+        <p>Note that the core profile is not exclusive. Device implementers are free to adopt 
```

Comment:
also see https://github.com/w3c/wot-profile/issues/21

Reply:
I wonder whether it makes sense to make a *technical* statement such as calling it according to the technology used e.g,. "http-json" profile (as far as I understand the recent discussion that's what it will be now, see https://github.com/w3c/wot-profile/issues/9#issuecomment-666171512. Correct? Different to the phase the  profile work started were the idea was to setup _means_ to define a profile).

Using actual names like "http-json" make the idea of a given profile very easy to grasp. Generic terms like "core" or "default" are very fuzzy with that regard.